data/json/furniture.json

```diff
@@ -1477,7 +1477,7 @@
     "type": "furniture",
     "id": "f_coffin_c",
     "name": "coffin",
-    "description": "Holds the bodies of the countless you kill.",
+    "description": "Holds the bodies of the countless killed in the Cataclysm.  The Zed, however, don't get buried.  They get pulped.",
```

Comment:
Technically you could bury a pulped zombie if you wanted, and there may be times you want to. I would take the second sentence out.

Reply:
I mainly didn't like how it suggests that I bury zombie bodies...which I've never done. Perhaps just "Holds the bodies of the countless killed in the Cataclysm."